core/shared/src/main/scala/newts/Newtype.scala

```diff
@@ -0,0 +1,18 @@
+package newts
+
+/**
+  * `S` is a newtype means `S` is a wrapper around an `A`
+  */
+trait Newtype[S, A] {
+  def wrap(value: A): S
+  def unwrap(value: S): A
+}
```

Comment:
Oh, but that then breaks Simulacrum in a different way, IIRC. Well, at least `Newtype` isn’t big – you can just manually write the `Ops` class with `unwrap`.